smart_control/reinforcement_learning/observers/rendering_observer.py

```diff
@@ -546,14 +546,15 @@ def __call__(self, trajectory: trajectory_lib.Trajectory) -> None:
       self._start_time = pd.Timestamp.now()
 
     if self._counter % self._render_interval_steps == 0 and self._environment:
-      logger.info(f'Rendering environment at step {self._counter}...')
+      logger.info('Rendering environment at step %d ...', self._counter)
       execution_time = pd.Timestamp.now() - self._start_time
       mean_execution_time = execution_time.total_seconds() / self._counter
 
       logger.info(
-          f'Step {self._counter}: '
-          f'Cumulative reward = {float(self._cumulative_reward):.2f}, '
-          f'Mean execution time = {mean_execution_time:.2f}s'
+          'Step %d: Cumulative reward = %.2f, Mean execution time = %.2fs',
```

Comment:
In the original logging statement `{mean_execution_time:.2f}s`, the 's' was to denote seconds, not to be used as a formatting directive. Confirming this new `'%.2fs` syntax will still work:

![Screenshot 2025-05-02 11 52 44 AM](https://github.com/user-attachments/assets/e46cbe47-3bc1-4cd7-917e-9fda5c8782f7)